README.rst

```diff
@@ -389,6 +389,29 @@ TMT_SHOW_TRACEBACK
     traceback is not printed out. When ``TMT_SHOW_TRACEBACK`` is
     set to any string except ``0``, traceback would be printed out.
 
+TMT_GIT_CREDENTIALS_URL\_, TMT_GIT_CREDENTIALS_VALUE\_
+    Prefixes of variable pairs used to provide credentials to clone git
```

Comment:
then we can maybe omit `Prefixes` from here?

Reply:
fixed in e7ac97ca